bindings/kitsune_runner/src/cli.rs

```diff
@@ -0,0 +1,73 @@
+use crate::common::get_connection_string;
+use clap::Parser;
+use wind_tunnel_runner::parse_agent_behaviour;
+use wind_tunnel_runner::prelude::{ReporterOpt, WindTunnelScenarioCli};
+
+#[derive(Parser)]
+#[command(about, long_about = None)]
+pub struct WindTunnelKitsuneScenarioCli {
+    /// The bootstrap server URL.
+    #[clap(long)]
+    pub bootstrap_server_url: String,
+
+    /// The signal server URL.
+    #[clap(long)]
+    pub signal_server_url: String,
+
+    /// The number of agents to run.
+    #[clap(long)]
+    pub agents: Option<usize>,
```

Comment:
This is a debatable option. If tests are running on the same machine, it makes sense. When tests are run on multiple machines, not so much.